### ISSUE TITLE
Modifieres in key mappings

### ISSUE THREAD

**Issue by dhelonious** [tsunamayo/Starship-EVO#2229]
### Short Description of the issue:

Key combinations (i.e. one or more modifiers + key) cannot be assigned to actions and movements in the key mapping options. However, there are key bindings with modifiers, which are not configurable and hence "hidden".

For example, the actions "Rotate Left" and "Rotate Right" are both assigned to R by default. However, "Rotate Left" requires the modifier LEFT SHIFT to rotate counter-clockwise. If "Rotate Right" is changed to another key, one can still use LEFT SHIFT + key to get a clockwise rotation (which is then the same as "Rotate Left").

I think this really confusing and could be resolved if the key mapping would work with something like this
```
RotateLeft=r
RotateRight=left shift+r
```
instead of
```
RotateLeft=r
RotateRight=r
```
with the "hidden modifier" LEFT SHIFT.

### Environment
- Starship EVO version: 20w22c

**Comment by tsunamayo** [comment]
Yes I wanted to do that, I wanted to get the hotfix out as there was a conflict for rotate. Thanks
